examples/Makefile.am

```diff
@@ -6,11 +6,11 @@
 #
 
 examplesdir = $(pkgdatadir)/examples
-dist_examples_DATA = \
+dist_examples_DATA =  \
 	ucx_hello_world.h \
 	ucp_hello_world.c \
 	uct_hello_world.c \
-	ucx_profiling.c \
+	ucx_profiling.c   \
```

Comment:
let's move ucx_profiling to test/apps, since it's not really an example

Reply:
moved